src/current/cockroachcloud/authorization.md

```diff
@@ -114,10 +120,32 @@ Cluster Creators can create clusters in an organization. A cluster's creator is
 
 ### Cluster Developer
 
-Users with this role can view cluster details and access the [DB Console](network-authorization.html#db-console), allowing them to [export a connection string from the cluster page UI](authentication.html#the-connection-string), although they will still need a Cluster Administrator to [provision their SQL credentials](managing-access.html#manage-sql-users-on-a-cluster) for the cluster.
+Cluster Developers can view cluster details and access the [DB Console](network-authorization.html#db-console), allowing them to [export a connection string from the cluster page UI](authentication.html#the-connection-string), although they will still need a Cluster Administrator to [provision their SQL credentials](managing-access.html#manage-sql-users-on-a-cluster) for the cluster.
 
 This role can be granted for specific clusters or for all clusters in the organization.
 
+### Folder Admin
+
+{% include_cached feature-phases/limited-access.md %}
+
+This role is available only when your organization is enrolled in the Folders Limited Access.
+
+Folder Admins can create, rename, and move, or delete folders where they are granted the role, and they can also manage access to these folders. This role can be granted at the level of the organization or on a specific folder. If granted on a specific folder, the role is inherited by descendent folders.
+
+A user with the [Cluster Administrator](#cluster-administrator) role can grant themselves, another user, or a service account the Folder Admin role.
```

Comment:
+1 to what Jose said. Folder Admins can grant any relevant roles on folders and clusters to any user including themselves (within their scope)

Reply:
+1 to the above.  One technicality though: while Cluster Admins can't grant anything on FOLDERS, they can grant FOLDER roles on CLUSTERS.  This ends up doing nothing though, since Folder roles don't mean anything on a cluster.  

There is a story tracking a fix for this (https://cockroachlabs.atlassian.net/browse/CC-25058).  Not sure if this distinction is worth mentioning to customers though.

Reply:
We shouldn't mention that distinction if it isn't an intended behavior.

Reply:
Done